<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>financeiro</class>
 <widget class="QDialog" name="financeiro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="leftMenu" native="true">
        <property name="styleSheet">
         <string notr="true">#leftMenu
{
	
	background-color: rgb(254, 191, 78);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="leftMenuSub" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="logo">
                 <property name="maximumSize">
                  <size>
                   <width>166</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#logo
{
	backgroud-repeat: no-repeat;
backgroud-position: center;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../imgplural/nova.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="homeButton">
                 <property name="toolTip">
                  <string>Home</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#homeButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
	margin-top: 30px
}</string>
                 </property>
                 <property name="text">
                  <string>Home</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/home.svg</normaloff>../imgplural/feather/home.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="finButton">
                 <property name="toolTip">
                  <string>Financeiro</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#finButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
	margin-top: 30px
}</string>
                 </property>
                 <property name="text">
                  <string>Financeiro</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/dollar-sign.svg</normaloff>../imgplural/feather/dollar-sign.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="admButton">
                 <property name="toolTip">
                  <string>Administrativo</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#admButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	margin: 0;
	text-align: left;
	padding: 2px 10px;
	margin-top: 30px
}</string>
                 </property>
                 <property name="text">
                  <string>Administrativo</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/archive.svg</normaloff>../imgplural/feather/archive.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="infoButton">
                 <property name="toolTip">
                  <string>Help</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#infoButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
}</string>
                 </property>
                 <property name="text">
                  <string>Informações</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/info.svg</normaloff>../imgplural/feather/info.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="helpButton">
                 <property name="toolTip">
                  <string>Help</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#helpButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
	margin-top: 20px
}</string>
                 </property>
                 <property name="text">
                  <string>Help</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/help-circle.svg</normaloff>../imgplural/feather/help-circle.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightMenu" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#rightMenu
{
	
	background-color: rgb(26, 35, 25);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>30</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#label_6
{
	text-align: center;
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Financeiro</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="extensivoAlunosBtn">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#extensivoAlunosBtn
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	margin: 0;
	text-align: center;
	
	color: rgb(255, 255, 255);
}</string>
                 </property>
                 <property name="text">
                  <string>Mensalidades</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/iconWhite/plus-circle.svg</normaloff>../imgplural/iconWhite/plus-circle.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>260</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="extensivoProfBtn">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#extensivoProfBtn
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	margin: 0;
	text-align: center;
	color: rgb(255, 255, 255);
	
}</string>
                 </property>
                 <property name="text">
                  <string>Pagamentos</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/iconWhite/minus-circle.svg</normaloff>../imgplural/iconWhite/minus-circle.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>260</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="extensivoHoraBtn">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#extensivoHoraBtn
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	margin: 0;
	text-align: center;
	
	color: rgb(255, 255, 255);
}



#extensivoHoraBtn:hover
{
	
	
	background-color: rgb(50, 50, 50);
}

</string>
                 </property>
                 <property name="text">
                  <string>Caixa</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/iconWhite/dollar-sign.svg</normaloff>../imgplural/iconWhite/dollar-sign.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>260</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frameCaixa">
                 <property name="styleSheet">
                  <string notr="true">#frameCaixa
{
	background-color: rgb(254, 191, 78);
	border-radius: 10px;
	margin-top: 20px;
	margin-left: 20px;
	margin-bottom: 20px;
	margin-right: 20px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QFrame" name="frame_4">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="font">
                        <font>
                         <pointsize>12</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>Entrada</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_7">
                       <property name="font">
                        <font>
                         <pointsize>12</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>Saída</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="font">
                        <font>
                         <pointsize>12</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>Total em Caixa</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_9">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <item>
                      <widget class="QLabel" name="labelEntrada">
                       <property name="font">
                        <font>
                         <pointsize>18</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">#label
{
	background-color: rgb(250, 250 ,250);
	border-radius: 10px;
	margin-top: 10px;
	margin-left: 10px;
	margin-bottom: 10px;
	margin-right: 10px;
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="labelSaida">
                       <property name="font">
                        <font>
                         <pointsize>18</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">#label_4

{
	background-color: rgb(250, 250 ,250);
	border-radius: 10px;
	margin-top: 10px;
	margin-left: 10px;
	margin-bottom: 10px;
	margin-right: 10px;
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="labelTotal">
                       <property name="font">
                        <font>
                         <pointsize>18</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">#label_2

{
	background-color: rgb(250, 250 ,250);
	border-radius: 10px;
	margin-top: 10px;
	margin-left: 10px;
	margin-bottom: 10px;
	margin-right: 10px;
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
